TuneUp/TuneUpViewExtension.cs

```diff
@@ -49,7 +49,7 @@ public void Loaded(ViewLoadedParams p)
                 }
 
             };
-            p.AddMenuItem(MenuBarType.View, TuneUpMenuItem);
+            p.AddExtensionMenuItem(TuneUpMenuItem);
```

Comment:
hmm can you test loading this view ex in legacy version of Dynamo? I think part of the AC is that if user download newer version of TuneUp in older version of Dynamo e.g. 2.6.1, Dynamo should not crash when loading TuneUp. This code may crash in older version of Dynamo, we may need to wrap this code in try catch and fall back to the older version of API

Reply:
Address this issue in the latest commit.